getting-started/dc2-guide.rst

```diff
@@ -0,0 +1,182 @@
+########################################
+Processing DC2 data with the Gen3 Butler
+########################################
+
+The data explored in this guide is simulated data, the same kind used for Rubin's Data Preview 0 (DP0).
+
+.. note::
+
+   This guide assumes the user has access to a shared "Gen3" Butler repository containing data from the Dark Energy Science Collaboration (DESC)'s Data Challenge 2 (DC2), most likely on the ``lsstdevl`` machines at NCSA.
+   This guide further assumes the user has a recently-built version of :py:mod:`lsst.distrib` from the LSST Science Pipelines.
+   Finally, this guide assumes the user is interested in running the Alert Production (AP) pipeline on this data using good seeing templates.
+
+What's in the Shared DC2 Butler Repository?
+===========================================
+
+* The instrument is called ``LSSTCam-imSim``
+* The obs-package is ``obs_lsst``, i.e., :py:mod:`lsst.obs.lsst` (note ``imsim`` subdirectories within)
+* The skymap is called ``DC2``
+* Patches go from 0 to 48
+* Detectors go from 0 to 188
+* Available bands are ``ugrizy``
+* Data exist for some tracts in the ~2553–5074 range
+* One set of reference catalogs are called ``cal_ref_cat_2_2``, and have some filtermap definitions that must be specified
+* On NCSA's ``lsstdevl`` in ``/repo/dc2``, tracts 3828 and 3829 are in the collection ``2.2i/defaults/test-med-1``
+* A smaller CI dataset, just patch 24 in tract 3828, is in ``2.2i/defaults/ci_imsim`` and corresponds to the GitHub repo ``testdata_ci_imsim``
+
+To see some collections, try, e.g.,
+
+.. prompt:: bash
+
+   butler query-collections /repo/dc2 --chains=tree "2.2i/*"
+   butler query-collections /repo/dc2 --chains=tree "u/mrawls/*"
+
+These commands should print a list of collections that meet the search criteria.
+The "tree chain" view is a handy way to see the "chained" data products that are included with each collection, such as calibs, skymaps, refcats, etc.
+
+To see some available datasets for processing, try, e.g.,
+
+.. prompt:: bash
+
+   butler query-data-ids /repo/dc2 tract patch visit --collections='2.2i/defaults/test-med-1' --where "skymap='DC2' AND band='g' AND tract=3828 AND patch=47 AND instrument='LSSTCam-imSim'"
+
+This command should print a list of data IDs that meet the search criteria, along with their tract, patch, and visit number.
+
```

Comment:
Could you include a comment stating which of the `--where` arguments are required?